cli/src/extract-tests/index.js

```diff
@@ -250,4 +263,22 @@ async function deleteExtraAddonFiles(analysis) {
     force: true,
     recursive: true,
   });
+
+  await packageJson.modify((json) => {
```

Comment:
there is also:
 - https://ember-apply.pages.dev/funcs/packageJson.removeDevDependencies
 - https://ember-apply.pages.dev/funcs/packageJson.removeDependencies
 
 but no remove scripts :upside_down_face: probably fine. and what you have is fine, too
 
 :100:

Reply:
Yeah, I know, just thought that given I already do `.modify()`, I can do everything in that place. But I have no string feelings about that...

Reply:
I don't either -- I left this comment before I got to the end of the function, and what exists as you wrote it is pretty important, so I'm happy